grafana-plugin/src/containers/ApiTokenSettings/ApiTokenSettings.tsx

```diff
@@ -67,6 +67,16 @@ class ApiTokens extends React.Component<ApiTokensProps, any> {
       },
     ];
 
+    const requiredPermissionToView = UserActions.APIKeysWrite;
```

Comment:
Do we really need a separate const for this instead of just referencing `UserActions.APIKeysWrite` ?

Reply:
addressed [here](https://github.com/grafana/oncall/pull/1185/commits/48cf7c3de02afd763f1fdf05cc0b9177c76eccaa) 👍